Extensions/xkit_patches.js

```diff
@@ -124,33 +124,74 @@ XKit.tools.parse_version = function(versionString) {
 	return version;
 };
 
+var blogs_get = document.createElement("script");
+blogs_get.textContent = "("+(function(){
+	var blogs = [];
+	var models = Tumblr.dashboardControls.allTumblelogs;
+
+	models.filter(function(model){
```

Comment:
At least in Firefox models.filter, models.forEach, models.map, etc. are all blocked as complex functions, whereas models[i] works.